test/rules/prefer-while/bad-with-condition/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "prefer-while": true
```

Comment:
tests that share the same config file should appear in the same `.ts.lint` file. please merge all of these so there's only `true` and `false` tests. one `.ts.lint` file can have more than one "test case" in it.

Reply:
Done in 8fa8062.  I added some comments to the test cases.  Please let me know if they look amiss or should be added in the first place.